CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+- Fixed the wrong path of a png file `/frotary` to `/frontary` in `theme.css`.
```

Comment:
```suggestion
- Fixed the wrong path of a delete-x.png file `/frotary` to `/frontary` in `theme.css`.
```

What do you think about making this more specific like above?

Reply:
I think `/` should be removed, but, other things are simply suggestions. It's totally up to you.

Reply:
The more specific, the better for all engineers. I've just fixed it as you figured out.